packages/ai-bot/lib/matrix.ts

```diff
@@ -60,12 +67,13 @@ export async function sendMessageEvent(
   commandRequests: Partial<CommandRequest>[] = [],
   reasoning: string | undefined = undefined,
 ) {
+  let html = markdownToHtml(escapeHtmlOutsideCodeBlocks(body));
```

Comment:
This is the actual root cause. The formatted message must be in html, but we didn't convert the body which is in markdown into html.